launcher/src/test/java/org/apache/brooklyn/launcher/CleanOrphanedLocationsIntegrationTest.java

```diff
@@ -49,7 +49,7 @@
 import java.util.Map;
 import java.util.Set;
 
-public class CleanOrphanedLocationsLiveTest {
+public class CleanOrphanedLocationsIntegrationTest {
```

Comment:
Tests are not in Integration group but since there is real data which is being used for the test may be I can consider it Integration. 
- I think it is good to have these tests for now, @aledsage wdyt?
- @aledsage do you think these tests should be refactored to use the library you have in `CleanOrphanedLocationsLiveTest`